WordPress/Classes/Utility/App Configuration/AppConstants.swift

```diff
@@ -20,6 +22,10 @@ import WordPressAuthenticator
     static let pushNotificationAppId = "org.wordpress.appstore"
     #endif
     #endif
+
+    static var currentAppIconName: String {
```

Comment:
I think this would be a better addition in `AppStyleGuide`: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Jetpack/AppStyleGuide.swift#L29

Reply:
Thanks! I pushed some code to make it easier to just get the current app icon from the `AppIcon` struct / helper, and I extracted that file from the AppIconViewController as it's now used in more than one place. Most of the code in there is just as it was, I just added a new method to fetch the current icon filename as a string and also some comments.